### PR TITLE
SONARPY-483 SONARPY-484 Rule S1144 and Rule S3985

### PR DESCRIPTION
**Pull request by andrea-guarino-sonarsource**

### CODE REVIEW

File: python-checks/src/test/resources/checks/unreadPrivateMethods.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class A:
```
Comment (guillaume-dequenne-sonarsource) [inline python-checks/src/test/resources/checks/unreadPrivateMethods.py]: Maybe we could have a test for an unused method with 1 leading underscore to ensure we don't report any issues on those.

File: python-checks/src/test/resources/checks/unreadPrivateMethods.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  __foo = 42 # OK, raised by S4487
+
+  def __unused(self): ... # Noncompliant {{Remove this unused class-private '__unused' method.}}
```
Comment (guillaume-dequenne-sonarsource) [inline python-checks/src/test/resources/checks/unreadPrivateMethods.py]: Super nitpicky but is there a reason why we name the method specifically in the message, yet we don't do it for classes in S3985?

Comment (andrea-guarino-sonarsource) [reply python-checks/src/test/resources/checks/unreadPrivateMethods.py]: Good question. @nicolas-harraudeau-sonarsource , wdyt?

Comment (andrea-guarino-sonarsource) [reply python-checks/src/test/resources/checks/unreadPrivateMethods.py]: For the record, I decided to edit S3985 message to include class name.

**Pull request merged**